sources/europe/de/Brandenburg-DOP20c.geojson

```diff
@@ -0,0 +1,1564 @@
+{
+  "type": "Feature",
+  "properties": {
+    "id": "Brandenburg-DOP20c",
+    "name": "Brandenburg GeoBasis-DE/LGB (2020-2022) / DOP20c",
+    "type": "wms",
+    "category": "photo",
+    "url": "https://isk.geobasis-bb.de/mapproxy/dop20c/service/wms?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=bebb_dop20c&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+    "best": true,
+    "start_date": "2020",
+    "end_date": "2022",
+    "country_code": "DE",
+    "description": "Digitale Orthophotos f\u00fcr das Brandenburg mit einer Bodenaufl\u00f6sung von 0,20 m. Regionen haben unterschiedlich alte Daten, vgl. https://data.geobasis-bb.de/geobasis/information/aktualitaeten/bb_dop_aktualitaet.pdf.",
+    "available_projections": [
+      "EPSG:25833",
+      "EPSG:25832",
+      "EPSG:4326",
+      "EPSG:4258",
+      "EPSG:3034",
+      "EPSG:3035",
+      "EPSG:3045",
+      "EPSG:3857",
+      "EPSG:4839",
+      "EPSG:5650"
+    ],
+    "attribution": {
+      "text": "GeoBasis-DE/LGB 2020 / Digitale farbige Orthophotos aktuell; Geoportal Berlin, dl-de/by-2-0",
+      "required": true
+    },
+    "license_url": "https://wiki.openstreetmap.org/wiki/Brandenburg/Geoportal#Rechtliche_Grundlagen",
+    "privacy_policy_url": "https://geobasis-bb.de/lgb/de/datenschutz/"
+  },
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
```

Comment:
👁️ I think we should use a simplfied version of this. The level of detail is not needed for data consumers but it does increase the overall size of ELI…

This is a simplfied version:

<img width="300" alt="Bildschirm­foto 2023-05-24 um 06 25 45" src="https://github.com/osmlab/editor-layer-index/assets/111561/9ea9b586-90b4-436e-ba12-e721daed2a27"> vs. <img width="300" alt="Bildschirm­foto 2023-05-24 um 06 25 38" src="https://github.com/osmlab/editor-layer-index/assets/111561/d93f4cf6-64ca-4720-9992-bc138b0d0f50">



```json
 [
      [
        [
          14.24812,
          53.44498
        ],
        [
          13.94777,
          53.44338
        ],
        [
          13.82374,
          53.56695
        ],
        [
          13.76178,
          53.56646
        ],
        [
          13.7634,
          53.49522
        ],
        [
          13.64291,
          53.47585
        ],
        [
          13.61447,
          53.42287
        ],
        [
          13.49525,
          53.38468
        ],
        [
          13.49734,
          53.31249
        ],
        [
          13.40775,
          53.31153
        ],
        [
          13.40887,
          53.27527
        ],
        [
          13.25828,
          53.2913
        ],
        [
          13.23163,
          53.23669
        ],
        [
          13.07997,
          53.25215
        ],
        [
          13.08134,
          53.21649
        ],
        [
          13.02337,
          53.19734
        ],
        [
          12.87539,
          53.19512
        ],
        [
          12.57093,
          53.279
        ],
        [
          12.45219,
          53.25885
        ],
        [
          12.45032,
          53.29401
        ],
        [
          12.32711,
          53.34561
        ],
        [
          12.02346,
          53.37484
        ],
        [
          11.99718,
          53.30262
        ],
        [
          11.93855,
          53.28312
        ],
        [
          11.52421,
          53.21711
        ],
        [
          11.55691,
          53.18073
        ],
        [
          11.53095,
          53.12677
        ],
        [
          11.2614,
          53.13771
        ],
        [
          11.26733,
          53.06413
        ],
        [
          11.56749,
          53.03688
        ],
        [
          11.93731,
          52.86675
        ],
        [
          12.20477,
          52.85535
        ],
        [
          12.23851,
          52.82045
        ],
        [
          12.18181,
          52.78489
        ],
        [
          12.21998,
          52.64172
        ],
        [
          12.16221,
          52.64048
        ],
        [
          12.14081,
          52.49513
        ],
        [
          12.25681,
          52.49761
        ],
        [
          12.28856,
          52.44569
        ],
        [
          12.25975,
          52.4451
        ],
        [
          12.26679,
          52.31871
        ],
        [
          12.21535,
          52.26395
        ],
        [
          12.24628,
          52.22921
        ],
        [
          12.19309,
          52.15475
        ],
        [
          12.37579,
          52.0146
        ],
        [
          12.55345,
          51.96412
        ],
        [
          12.7272,
          51.985
        ],
        [
          12.8457,
          51.95175
        ],
        [
          12.84718,
          51.91624
        ],
        [
          13.11066,
          51.86696
        ],
        [
          13.14611,
          51.68832
        ],
        [
          13.0314,
          51.66872
        ],
        [
          13.03277,
          51.63183
        ],
        [
          13.18141,
          51.54377
        ],
        [
          13.15526,
          51.50878
        ],
        [
          13.18681,
          51.38342
        ],
        [
          13.41714,
          51.42155
        ],
        [
          13.53469,
          51.3522
        ],
        [
          14.02539,
          51.35615
        ],
        [
          14.10818,
          51.51766
        ],
        [
          14.16491,
          51.53709
        ],
        [
          14.36752,
          51.50248
        ],
        [
          14.53943,
          51.55703
        ],
        [
          14.71254,
          51.53843
        ],
        [
          14.71253,
          51.57579
        ],
        [
          14.79936,
          51.59343
        ],
        [
          14.79951,
          51.68525
        ],
        [
          14.68303,
          51.73902
        ],
        [
          14.68297,
          51.81077
        ],
        [
          14.6526,
          51.81075
        ],
        [
          14.74142,
          51.89881
        ],
        [
          14.74145,
          51.98926
        ],
        [
          14.79679,
          52.04359
        ],
        [
          14.76766,
          52.11645
        ],
        [
          14.70883,
          52.11646
        ],
        [
          14.73768,
          52.2607
        ],
        [
          14.58963,
          52.33182
        ],
        [
          14.58935,
          52.42109
        ],
        [
          14.67665,
          52.4749
        ],
        [
          14.64846,
          52.54703
        ],
        [
          14.67652,
          52.58388
        ],
        [
          14.40937,
          52.72722
        ],
        [
          14.37872,
          52.7806
        ],
        [
          14.16964,
          52.83346
        ],
        [
          14.19761,
          52.90458
        ],
        [
          14.16827,
          52.9401
        ],
        [
          14.40699,
          53.04797
        ],
        [
          14.43217,
          53.1581
        ],
        [
          14.40263,
          53.193
        ],
        [
          14.49102,
          53.24588
        ],
        [
          14.4603,
          53.33826
        ],
        [
          14.31001,
          53.31813
        ],
        [
          14.25015,
          53.26497
        ],
        [
          14.16377,
          53.28335
        ],
        [
          14.24936,
          53.33533
        ],
        [
          14.24812,
          53.44498
        ]
      ]
    ]
```

Reply:
Change applied in https://github.com/osmlab/editor-layer-index/pull/2109